src/app/Scenes/MyCollection/Components/MyCollectionArtistFiltersStickyTab.tsx

```diff
@@ -0,0 +1,43 @@
+import { Flex, Input, useSpace } from "@artsy/palette-mobile"
+import SearchIcon from "app/Components/Icons/SearchIcon"
+import { MyCollectionArtworksKeywordStore } from "app/Scenes/MyCollection/Components/MyCollectionArtworksKeywordStore"
+import { debounce } from "lodash"
+import { useEffect, useState } from "react"
+
+export const MyCollectionArtistFilters: React.FC<{}> = () => {
+  const space = useSpace()
+  const keyword = MyCollectionArtworksKeywordStore.useStoreState((state) => state.keyword)
+  const [query, setQuery] = useState(keyword)
+
+  const setKeyword = debounce(
+    MyCollectionArtworksKeywordStore.useStoreActions((actions) => actions.setKeyword)
+  )
```

Comment:
Don't we need to pass a delay to `debaunce`?

```suggestion
  const setKeyword = debounce(
    MyCollectionArtworksKeywordStore.useStoreActions((actions) => actions.setKeyword)
  )
```

Reply:
the default time was working well that's why I didn't add. but it only makes sense to add it for consistency. I will update

Reply:
Up to you whether you want to use the default or pass a delay. :)